neo4j/activity/executeCypherQuery/metadata.go

```diff
@@ -0,0 +1,56 @@
+package executeCypherQuery
+
+import (
+	"github.com/project-flogo/core/data/coerce"
+)
+
+// Settings structure
+type Settings struct {
+	Connection   string `md:"connection,required"` // The Neo4j connection
+	DatabaseName string `md:"databaseName,required"`
+	AccessMode   string `md:"accessMode,required"`
+}
+
+//Input structure
+type Input struct {
+	CypherQuery string                 `md:"cypherQuery,required"` // The cypher query
+	QueryParams map[string]interface{} `md:"queryParams"`
+}
+
+//Output structure
+type Output struct {
+	Output interface{} `md:"response"` // The JSON Response of the query
+}
+
+//FromMap method
+func (i *Input) FromMap(values map[string]interface{}) error {
+	i.CypherQuery, _ = coerce.ToString(values["cypherQuery"])
```

Comment:
Only a nitpick, I think you should not ignore error here.

Reply:
Yes, you should probably just return the error if it is not nil.
```
  var err error
  i.CypherQuery, err = coerce.ToString(values["cypherQuery"])
  if err != nil {
    return err
  }
```